addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/internal/ColorUtil.java

```diff
@@ -0,0 +1,91 @@
+package org.openhab.binding.mihome.internal;
```

Comment:
Please add proper [copyright comment](https://github.com/openhab/openhab2-addons/pull/1738/files#diff-8df51d9e6a9c7447d78093d0d3f01f4dR2) in each class file.